lib/jsdom/living/nodes/HTMLDialogElement-impl.js

```diff
@@ -1,8 +1,123 @@
 "use strict";
-
+const DOMException = require("domexception/webidl2js-wrapper");
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
+const { fireAnEvent } = require("../helpers/events");
+const { runAutoFocus, outsideUnopenedDialog } = require("../helpers/focusing");
+
+const ALREADY_OPEN = "Failed to execute 'showModal' on 'HTMLDialogElement': " +
+                    "The element already has an 'open' attribute, and therefore cannot be opened modally.";
+
+const NOT_CONNECTED = "Failed to execute 'showModal' on 'HTMLDialogElement': " +
+                      "The element is not in a Document.";
+
+class HTMLDialogElementImpl extends HTMLElementImpl {
+  constructor(globalObject, args, privateData) {
+    super(globalObject, args, privateData);
+    this._isModal = false;
+    this._returnValue = "";
+    this._previousFocusElement = null;
+  }
+
+  get open() {
+    return this.getAttributeNS(null, "open") !== null;
+  }
+
+  get returnValue() {
```

Comment:
The getters and setters for returnValue should not be necessary. Just rename the field from `_returnValue` to `returnValue`.